app/src/components/explorer/SearchResultsTable.vue

```diff
@@ -0,0 +1,293 @@
+        </div>
+      </div> -->
+
+      <!-- New -->
+      <!-- <div :class="(resultsTab =='getImages' || resultsTab== 'getCharts') ? 'grid_explorer-boxes' : 'grid_explorer-fullrow'">
+        <div v-for="(result, index) in getArticles"
+          :key="index"
+          class="btn--animated md-card gallery-item results_card">
+
+          <div class="utility-gridicon_explorer" v-if="resultsTab === 'getCharts'">
+            <div
+              @click.prevent="bookmark(result.name, true)"
+              v-if="result.bookmark"
+            >
+              <md-icon>bookmark</md-icon>
+            </div>
+            <div
+              @click.prevent="bookmark(result.name, false)"
+              v-else
+            >
+              <md-icon>bookmark_border</md-icon>
+            </div>
+          </div>
+
+          <md-card-media-cover
+            v-if="resultsTab=='getImages' || resultsTab== 'getCharts'"
+            md-solid
+          >
+            <md-card-media md-ratio="4:3"  v-if="result.thumbnail">
+              <img
+                :src="getThumbnailUrl(result)"
+                :alt="result.label"
+                v-if="result.thumbnail"
+              >
+            </md-card-media>
+            <md-icon v-else class="md-size-5x"> image </md-icon>
+          </md-card-media-cover>
+
+          <md-card-header style="padding:0px">
+            <md-avatar v-if="resultsTab=='Samples' && result.thumbnail">
+              <img
+                :src="getThumbnailUrl(result)"
+                :alt="result.label"
+                v-if="result.thumbnail"
+              >
+            </md-avatar>
+            <router-link @click.native="fixUriBeforeRouting(result.identifier)" to="#"
+            class="results_card-title" v-if="resultsTab === 'getArticles'">
+              <div >{{ result.label }}</div>
+            </router-link>
+
+            <router-link @click.native="fixUriBeforeRouting(result.identifier)" to="#"
+            class="results_card-title" v-if="resultsTab === 'getSamples'">
+              <div >{{ result.label }}</div>
+            </router-link>
+
+            <router-link :to="`/explorer/chart/view/${result.identifier}`"  class="results_card-title">
+              <div >{{ result.label }}</div>
+            </router-link>
+            <div v-if="resultsTab !== 'getImages' && resultsTab !=='getCharts'">
+              <div class="results_card-type">{{resultsTab}}</div>
+              <div class="md-body-1 results_card-description" v-if="result.description" >
+                  {{ reduceDescription(result.description) }}
+              </div>
+              <div class="md-body-1 results_card-description" v-else-if="result.identifier" >
+                  {{ result.identifier }}
+              </div>
+            </div>
+          </md-card-header>
+        </div>
+      </div> -->
+      <!-- <pagination
+        :cpage="page"
+        :tpages="totalPages"
+        @go-to-page="loadItems($event)"
+      /> -->
```

Comment:
we should delete this instead of commenting out

Reply:
That content is still needed but we don't have a backend logic for it yet, hence the commenting out. The getCharts there is just a place-holder.